dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -146,14 +147,16 @@ export default defineComponent({
         const apiResponse = await metaDataControllerApi.getListOfDataMetaInfo(this.companyID);
         const listOfDataMetaInfoForCompany = apiResponse.data;
         const listOfDataIdsToEmit = [] as string[];
-        listOfDataMetaInfoForCompany.forEach((dataMetaInfo) => {
-          if (ARRAY_OF_FRONTEND_INCLUDED_FRAMEWORKS.includes(dataMetaInfo.dataType)) {
-            this.appendDistinctDataTypeToDropdownOptionsIfNotIncludedYet(dataMetaInfo.dataType);
-          }
-          if (dataMetaInfo.dataType === this.dataType) {
-            listOfDataIdsToEmit.push(dataMetaInfo.dataId);
-          }
-        });
+        listOfDataMetaInfoForCompany
+          .filter((dataMetaInfo) => dataMetaInfo.qaStatus == QAStatus.Accepted)
```

Comment:
So seemingly the backend returns even datasets that did not pass QA yet. 
This is fine - but we need to make sure, that only such datasets are returned with QA=Pending that this user  created.

Reply:
WIP

Reply:
We could also (or only because the data endpoints use the metadatamanager) put the filter on the metadata endpoints and the companies endpoint. But I would think that it is fine if any user can see that there is a pending or so and so many pending datasets on that company?

Reply:
I'll add a test once we have agreed on one strategy.

Reply:
Unit test implemented

Reply:
ok